README.md

```diff
@@ -24,60 +24,59 @@ The game ends when both players pass.
 
 As in the [CGOS](http://www.yss-aya.com/cgos/) rules, we will use [area scoring](<https://en.wikipedia.org/wiki/Go_(game)#Scoring_rules>). So a player's score is the number of stones they have on the board + the number of empty territories they surround at the end of the game.
 
-The score also includes a [Komi](<https://en.wikipedia.org/wiki/Go_(game)#Komi>) which is a handicap given to black as it is advantageous to lay the first stone. We will use a komi of 7.5 which is subtracted from black's score at the end of the game.
+The score also includes a [Komi](<https://en.wikipedia.org/wiki/Go_(game)#Komi>) which is a handicap given to black as it is advantageous to lay the first stone. We will use a komi of `7.5` which is subtracted from black's score at the end of the game.
 
 # Competition Rules :scroll:
 
 1. You must build an agent to play Go using either a **reinforcement learning** algorithm or a **heuristic search algorithm** (such as monte carlo tree search :deciduous_tree:) or a mix of **both**!
 
 2. You can only write code in `main.py`
 
+   - Your choose_move function will have a limit of 1 second to run!
    - You can only store data to be used in a competition in a `.pkl` file by `save_file()`.
-   - You can pkl anything you want, even a dictionary of pytorch networks! Just make sure your choose_move can read it.
+   - You can pkl anything you want, even a dictionary of pytorch networks (or nothing)! Just make sure your choose_move can read it.
    - In the competition, your agent will call the `choose_move()` function in `main.py` to select a move (`choose_move()` may call other functions in `main.py`)
```

Comment:
I think we should explain that we are initialising their `MCTS` class in the competition, so they cannot add arguments to it.